### PR TITLE
Move from master -> main

### PR DESCRIPTION
**Pull request by DrewScoggins**

### CODE REVIEW

File: scripts/dotnet.py [hunk 1]
```diff
@@ -854,7 +854,7 @@ def __process_arguments(args: list):
         dest='channels',
         required=False,
         nargs='+',
-        default=['master'],
+        default=['main'],
```
Comment (billwert) [inline scripts/dotnet.py:857]: This is the channel for the `dotnet-install` script, yes? Did they also change that?

Comment (DrewScoggins) [reply scripts/dotnet.py:857]: I added a new channel mapping that goes from main -> tfm: net6.0, channel: master

**Review by adamsitnik — APPROVED** [review]
The microbenchmarks part LGTM, thank you for working on this @DrewScoggins !

**Review by billwert — APPROVED** [review]

**Comment by billwert** [comment]
@DrewScoggins is this mergable?

**Comment by DrewScoggins** [comment]
I need to rip out the changes to the bootstrap code. I will do that and then we can merge.

**Pull request merged**